index.js

```diff
@@ -274,7 +277,9 @@ Polyglot.prototype.locale = function (newLocale) {
 //
 // This feature is used internally to support nested phrase objects.
 Polyglot.prototype.extend = function (morePhrases, prefix) {
-  forEach(morePhrases, function (phrase, key) {
+  forEach(entries(morePhrases), function (entry) {
```

Comment:
One issue I am noticing while trying this new version is that `for-each` would gracefully handle `null` and `undefined` here, but `object.entries` does not, causing errors like:

```
TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
```

I suppose we should change lines like this to:

```diff
-  forEach(entries(morePhrases), function (entry) {
-  forEach(entries(morePhrases || {}), function (entry) {
```

or similar (e.g. conditional with early return).

@ljharb thoughts?

Reply:
https://github.com/airbnb/polyglot.js/pull/161